### DISCUSSION TITLE
Login in Lychee 5.1.2 installed from Yunohost doesn't work

### DISCUSSION THREAD

**Discussion by KoRnBoy82** [LycheeOrg/Lychee#2272] [Bugs] [answered]
### Detailed description of the problem [REQUIRED]
*A clear and concise description of what the bug is.*

### Steps to reproduce the issue
**Steps to reproduce the behavior:**
1. Install Lychee from Yunohost store.
2. Installed DB inside phpMyAdmin.
3. Yunohost already has gnix so webserver is working by default

**Screenshots**
![image](https://github.com/LycheeOrg/Lychee/assets/22198808/59a9f9d6-12a5-42fe-bd04-f36832efdf6f)

### Output of the diagnostics  [REQUIRED]
I can't find the diagnostic required. Is there another mandatory service needed? Lychee was working as it before 5.X update un Yunohost.

### Browser and system
Microsoft Edge, Google Chrome,

**Comment by ildyria** [comment]
This is not related to the v5 because the v5 is only a change in the front-end. I am pretty sure it is related to the backend.
you can access the diagnostics by going to `yoururl.tld/diagnostics` or `yoururl.tld/Diagnostics`. You don't need to be logged in for that.

**Comment by KoRnBoy82** [comment]
I know that every Yunohost users are actually living this. The diagnostics is currently loading. We'll see.
![image](https://github.com/LycheeOrg/Lychee/assets/22198808/7c2dfa56-5934-4902-ae65-13242d73be69)

**Comment by ildyria** [comment]
Try with **`Diagnostics`** in the url, not `diagnostics` (uppercase D makes a difference).

**Comment by KoRnBoy82** [comment]
Still loading. Is this supposed to be long before displaying?
![image](https://github.com/LycheeOrg/Lychee/assets/22198808/f1a8ea17-75f2-4328-91d3-677cbc2b7ba7)

**Comment by ildyria** [comment]
Still wrong page...

What you should see:
![image](https://github.com/LycheeOrg/Lychee/assets/627094/4f1d26ed-b7f2-406c-9811-793d2f4c9f54)

**Comment by ildyria** [comment]
Also, to clarify, are you using : https://yunohost.org/en/app_lychee ?

**Comment by KoRnBoy82** [comment]
> Also, to clarify, are you using : https://yunohost.org/en/app_lychee ?

Yes. I used the app in the Yunohost store. Here is my setup in Yunohost. I just removed the domain part but I'm a french user.

![image](https://github.com/LycheeOrg/Lychee/assets/22198808/7c282708-0313-4e2a-9b82-123d86547887)

**Comment by KoRnBoy82** [comment]
So my URL should be https://domain.com/lychee/Diagnostics

**Comment by ildyria** [comment]
Well then you will have to see what is going wrong with them. Because this is out of our scope.
It seems that they need to fix things on their hand. See [here](https://github.com/YunoHost-Apps/lychee_ynh/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).

Our releases not are rather clear on that.

> So my URL should be [domain.com/lychee/Diagnostics](https://domain.com/lychee/Diagnostics)

That would be correct.

It also seems you are using sub directory setup. I have no clue if Yunohost did their change properly but they should read our [release notes](https://lycheeorg.github.io/docs/releases.html#v503)...

**Comment by KoRnBoy82** [comment]
I still have access to manual install with the command line tool. I will try to uninstall it and install using your guide. Thanks anyway. Have a nice day.

**Comment by ildyria** [comment, marked as answer]
I will unfortunately have to close this one because there is nothing I can do to help you.
Lychee is working fine in all the dockers installs (our image and LinuxServer), local installations, synology installations, with multiple DB backend.

### This is completely on YunoHost side to fix.

**Comment by ildyria** [comment]
Make sure you are running **php 8.2**
